### PR TITLE
Add loading for atom component used in the table

### PR DESCRIPTION
**Pull request by pochretien**
Add the behaviour for isLoading and disabled for each component used in the table for the new table loading

### CODE REVIEW

File: packages/react-vapor/src/components/navigation/pagination/tests/NavigationPaginationReducers.spec.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as _ from 'underscore';
 import {IReduxAction} from '../../../../utils/ReduxUtils';
 import {filterThrough} from '../../../filterBox/FilterBoxActions';
 import {selectListBoxOption} from '../../../listBox/ListBoxActions';
-import {TableHOCUtils} from '../../../table-hoc/TableHOCUtils';
+import {TableHOCUtils} from '../../../table-hoc/utils/TableHOCUtils';
```
Comment (gdostie) [inline packages/react-vapor/src/components/navigation/pagination/tests/NavigationPaginationReducers.spec.ts]: Unused import?
```suggestion
```

File: packages/react-vapor/src/components/table-hoc/reducers/TableRowReducers.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ import {
     ITableRowToggleCollapsiblePayload,
     TableHOCRowActionsType,
 } from '../actions/TableHOCRowActions';
-import {TableHOCUtils} from '../TableHOCUtils';
+import {TableHOCUtils} from '../utils/TableHOCUtils';
```
Comment (gdostie) [inline packages/react-vapor/src/components/table-hoc/reducers/TableRowReducers.ts]: Unused import?
```suggestion
```

File: packages/react-vapor/src/components/actions/ActionBarReducers.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ import {LoadingActions} from '../loading/LoadingActions';
 import {PaginationActions} from '../navigation/pagination/NavigationPaginationActions';
 import {PerPageActions} from '../navigation/perPage/NavigationPerPageActions';
 import {TableHOCRowActionsType} from '../table-hoc/actions/TableHOCRowActions';
-import {TableHOCUtils} from '../table-hoc/TableHOCUtils';
+import {TableHOCUtils} from '../table-hoc/utils/TableHOCUtils';
```
[the remark this answers is not in the document]
Comment (pochretien) [reply packages/react-vapor/src/components/actions/ActionBarReducers.ts]: only missing from the other one

File: packages/react-vapor/src/components/actions/tests/actionBarsReducers.spec.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import {selectListBoxOption} from '../../listBox/ListBoxActions';
 import {turnOffLoading, turnOnLoading} from '../../loading/LoadingActions';
 import {changePage} from '../../navigation/pagination/NavigationPaginationActions';
 import {changePerPage} from '../../navigation/perPage/NavigationPerPageActions';
-import {TableHOCUtils} from '../../table-hoc/TableHOCUtils';
+import {TableHOCUtils} from '../../table-hoc/utils/TableHOCUtils';
```
[the remark this answers is not in the document]
Comment (pochretien) [reply packages/react-vapor/src/components/actions/tests/actionBarsReducers.spec.ts]: used line 197

File: packages/react-vapor/docs/src/components/ComponentsInterface.tsx [hunk 5]
```diff
@@ -6,6 +6,11 @@ export interface IComponent {
     component: ExampleComponent;
 }
 
+export interface IComponentBehaviour {
```
[the remark this answers is not in the document]
Comment (pochretien) [reply packages/react-vapor/docs/src/components/ComponentsInterface.tsx]: moved

**Pull request merged**